src/components/infinite-scroll/__tests__/InfiniteScroll.test.tsx

```diff
@@ -16,154 +13,108 @@ const propsList: InfiniteScrollProps = {
     useWindow: true,
     onLoadMore: mockOnLoadMore,
     threshold,
-};
-
-const getSentinel = () => {
-    const nodes = document.querySelectorAll('[data-testid="sentinel"]');
-    return nodes[nodes.length - 1];
+    throttle: 0, // tests run too fast - keep it 0 for sanity
```

Comment:
what does it mean the tests are running too fast?

Reply:
So what happens is that the function doesn't end up getting called because the `scroll` event is instantaneous. Some of the tests take like 3-4 ms to execute, so even having the throttle be 1ms causes inconsistency.

Reply:
ahhh gotcha. this is so the function isn't throttled between separate tests, right? i.e. the first test will complete very quickly but when the second test runs, the `onContainerScroll` is still on "cool down" (due to the throttle and the first test completing fast)

Reply:
I think we can improve this comment to provide more clarity. maybe we could include how the 0 is to prevent throttling between tests?

Reply:
Actually, this was causing throttling within the test. I realized that we don't need this anymore tho - this was needed previously because the `onContainerScroll` function that loads content until the page becomes scrollable was sharing the same throttle. That's also one of the reasons I removed the throttle here:

https://github.com/box/box-ui-elements/pull/3057#discussion_r981809121